### ISSUE TITLE
Overview should be updated to Info

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#583]
As per new mocks, Topic Overview Tab has been updated to Topic Info Tab,

<img width="363" alt="Screenshot 2020-01-03 at 12 11 29 PM" src="https://user-images.githubusercontent.com/9396084/71710439-3fa89500-2e22-11ea-83c9-a5564e000459.png">

So, all files related to that needs to be updated like package name, file names, test cases names, xml files names, variable names, strings.xml, drawables, etc.

In short refactor `Overview` with `Info`.

**Comment by rt4914** [comment]
Closed via #587
